File: 241027/등장횟수 확인/check-number-of-appearances.cpp
#include <iostream>
using namespace std;

int main() {
    int cnt = 0;
    int a;
    
    for (int i = 0; i < 5; i++) {
        cin >> a;
        if (a % 2 == 0) {
            cnt++;
        }
    }

    cout << cnt;
    return 0;
}

File: 241027/등장횟수 확인/check-number-of-appearances.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split("\n");

let cnt = 0;

for (let i = 0; i < 5; i++) {

    let x = Number(input[i]);

    if (x % 2 === 0) {
        cnt++;
    }
}

console.log(cnt);